Source/SteveSloth/Public/MyPlayer.h

```diff
@@ -118,24 +119,33 @@ class STEVESLOTH_API AMyPlayer : public ACharacter
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, meta = (RowType = "MyAmmoData"), Category = "Data")
 	FDataTableRowHandle AmmoDataTable;
 
+public: // GETTERS/ACCESSORS
+	float GetMaxHealth() { return MaxHealth; }
+	float GetCurrentHealth() { return CurrentHealth; }
+	int GetEquippedAmmoIndex() { return EquippedAmmoIndex; }
+
+public: // SETTERS/MUTATORS
+	void SetMaxHealth(float amount) { MaxHealth = MaxHealth + amount; }
+	void SetCurrentHealth(float amount) { CurrentHealth = CurrentHealth + amount; }
+
```

Comment:
why are these moved here? shouldnt the getters & setters be right above the constructor

Reply:
this is because i messed up when merging from main.. ill bring them back to where they belong